docs/en/week12/12-2.md

```diff
@@ -0,0 +1,290 @@
+
+$$\mathcal{L}(\theta)=-\log p(y \mid x ; \theta)$$
+
+We can regularize the model using:
+- Dropout
+- Label Smoothing
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure2_1.png" style="background-color:#DCDCDC;" /><br>
+<b>Figure 2:</b> Supervised Learning from English to Nepali
+</center>
+
+Beginning with a supervised learning setting, we can use the aforementioned example of translation. Our dataset is
+
+$$D = \{(x,y)_i\}_{i=1, \vect{...}, N}$$
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure2_2.png" style="background-color:#DCDCDC;" /><br>
+<b>Figure 2:</b> Supervised Learning Visualization
+</center>
+
+We can train this model using maximum likelihood, where we want to maximize $\{ y\vert x \} $.
+
+One way to represent this methodology is by the diagram, where you have a blue encoder that processes English sentences, and a red decoder that processes Nepali. From here, we want to compute our cross-entropy loss and update our model parameters. We may also want to regularize the model using either dropout or label smoothing. Additionally, we may need to regularize using the log loss
+
+$$-\log p(y|x; \theta )$$
+
+*Note: Transliteration is not word per word translation. It means that you're using the charaters from one language to make a logical translation of the word in a different language.*
+
+**How can we improve generalization?**
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure3_1.png" style="background-color:#DCDCDC;" /><br>
+<b>Figure 3:</b> Semi-Supervised Learning (DAE) from English to Nepali
+</center>
+
+We can get additional source side monolingual data with this approach as well.
+
+$$
+\begin{array}{c}
+\mathcal{D}=\left\{(x, y)_{i}\right\}_{i=1, \ldots, N} \\
+\mathcal{M}^{s}=\left\{x_{j}^{s}\right\}_{j=1, \ldots, M_{s}}
+\end{array}
+$$
+
+When using the DAE learning framework, we an either pre-train or add a DAE loss to the supervised cross entropy term:
+
+$$
+\mathcal{L}^{D A E}(\theta)=-\log p(x \mid x+n)
+$$
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure3_2.png" style="background-color:#DCDCDC;" /><br>
+<b>Figure 3:</b> Semi-Supervised Learning (DAE) Diagram
+</center>
+
+One way to leverage an additional dataset is by trying to model $p(x)$ with a semi-supervised approach. One way to model $$p(x)$$ is via the denoising of an auto-encoder. This is particularly useful because the encoder and decoder share a similar machine translation methodology. In the semi-supervised learning approach, going from English to Nepali, we have:
+
+$$\mathcal{D} =\lbrace {(x, y)_{i} \rbrace }_{i=1, \vect{...}, N}$$
+
+$$\mathcal{M}^{s}=\left\{x_{j}^{s}\right\}_{j=1, \vect{...}, M_{s}}$$
+
+And when we want to predict missing labels using decoding and training noise, we can use cross-entropy loss as such:
+
+$\mathcal{L}^{S T}(\theta)=-\log p(\bar{y} \mid x+n)$
+
+$\mathcal{L}(\theta)=\mathcal{L}^{\sup }(\theta)+\lambda \mathcal{L}^\text{S T}(\theta)$
```

Comment:
Maybe you wanted to use a doulble `$`?
Also, use `\text{ST}` both times, and without a space between S and T.